books/free-programming-books-langs.md

```diff
@@ -824,6 +824,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Go Tutorial](http://www.tutorialspoint.com/go/) - Tutorials Point (HTML, PDF)
 * [Go Web Development Succinctly](https://www.syncfusion.com/succinctly-free-ebooks/go-web-development) - Mark Lewin (PDF, EPUB, Kindle)
 * [Golang by Example](https://golangbyexample.com)
+* [Golang Tutorial](https://golangbot.com/learn-golang-series/)
```

Comment:
```suggestion
* [Golang tutorial series](https://golangbot.com/learn-golang-series/) - Naveen Ramanathan
```
Please add author names where appropriate.
Don't invent titles! Use the title given by the resource.